testing/scenario_app/bin/run_ios_tests.dart

```diff
@@ -358,14 +358,23 @@ final _skipTestsForImpeller = [
   'ScenariosUITests/MultiplePlatformViewsTest/testPlatformView',
   'ScenariosUITests/NonFullScreenFlutterViewPlatformViewUITests/testPlatformView',
   'ScenariosUITests/PlatformViewMutationClipPathTests/testPlatformView',
+  'ScenariosUITests/PlatformViewMutationClipPathMultipleClipsTests/testPlatformView',
```

Comment:
Oh I didn't realize these were skipped for impeller... https://github.com/flutter/flutter/issues/131888#issuecomment-2327112432